src/styles/blocks/clock.scss

```
@@ -0,0 +1,96 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+html {
+  font-size: 44px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  margin: 0;
+  height: 100vh;
+  background-color: #FFEEDB;
+  @extend %centre-flex;
+}
+
+.watch {
+  @extend %watch-size;
+  @extend %watch-radius;
+  position: relative;
+  background-color: #f9dc5c;
+  border: 16px solid #558564;
+
+  &__centre {
+  @extend %centre-flex;
+  @extend %watch-centre;
+  @extend %watch-radius;
+  flex-direction: column;
+  height: calc(#{$watch-size / 20});
+  width: calc(#{$watch-size / 20});
```

redundant braces + duplication, you could have moved this value to a variable
```suggestion
  height: calc($watch-size / 20);
  width: calc($watch-size / 20);
```